addons/ai_hearing/functions/fnc_onSettingChanged.sqf

```diff
@@ -1,24 +1,23 @@
 #include "script_component.hpp"
 
 /*
-    Name: TFAR_ai_hearing_fnc_onSettingChanged
-
-    Author(s):
-        Dedmen
-
-    Description:
-        initializes/deinitializes the AI-Hearing
-        This function should only be called via CBA_Settings
-
-    Parameters:
-        BOOL: enable the setting
-
-    Returns:
-        NOTHING
-
-    Example:
-        true call TFAR_ai_hearing_fnc_onSettingChanged;
-*/
+ * Name: TFAR_ai_hearing_fnc_onSettingChanged
+ *
```

Comment:
Are these * on every line really needed? IMO that's annoying

Reply:
Currently, yes.